analyses/mwi/dryspells_trigger/exploration/2021_2022_postseason_overview.Rmd

```diff
@@ -0,0 +1,570 @@
+
+(1) its exclusive focus on January and February;
+(2) the use of low rainfall over a period as a definition of a dry spell;
+(3) its maximum rainfall amount over a period allowed to qualify as a dry spell.
+
+### Q1: Could the trigger have covered December and March dry spells?
+
+The Anticipatory Action (AA) trigger focused exclusively on January and February dry spells. Historically, dry spells have occurred in December and March however those months were not monitored due to the difficulty in assessing in real-time whether a dry period during those months was a dry spell or a streak of dry days outside of the rainy season. The 2021-2022 trigger assumed that the rainy season would have begun over the Southern Region by 24 Dec.
+
+The data for 2021-2022 confirms the challenge: while `r perc_nov_onset`% of the Southern Region began its rainy season in November, the remaining saw their season begin in December and as late as `r format(as.Date(onset_max, origin = "2021-11-01"), format = "%d %b")`.In detail, 50% of onset dates occurred by `r format(as.Date(onset_50perc_known, origin = "2021-11-01"), format = "%d %b")` and 90%  by `r format(as.Date(onset_90perc_known, origin = "2021-11-01"), format = "%d %b")`. Due to the need for 40 days to pass after the onset date to confirm that the season has begun, it took until `r format(as.Date(onset_max + 30, origin = "2021-11-01"), format = "%d %b")` to ascertain the onset for the entire region. Until then, it would not have been possible to confirm whether a dry period was an in-season dry spell for `r 100 - perc_nov_onset`% of the region. 
+
+As for cessation dates, the Southern Region saw its rainy season end at various times. Specifically, `r mar_cessation_area_pct`% of the Southern Region saw its rainy season end in March (mainly in the northern and southern tips). Any dry period before 15 March by definition must have been during the rainy season (since rainy season cannot end before 15 March) and therefore could be a dry spell. Dry periods starting on or after 15 March could have been during or after the rainy season. It would not have been possible to confirm before April whether the season had ended as the definition requires that a 15-day period after 15 March have passed. In other words, the earliest confirmation could only be made after 31 March. The implication is that the observational trigger could be expanded to cover up to 15 March as those dates are surely during the rainy season. Operationally, expansions beyond 15 March would require a time lag of 15 days after the 14th day of a suspected dry spell to verify that it is an in-season dry spell. However, it is likely that most dry periods suspected of being dry spells would end up also meeting the criterion for rainy season cessation and therefore be outside the season. How frequently that may be the case can be verified empirically using historical data.
```

Comment:
Interesting. Makes me wonder if instead of having a fixed rainy season cessation date, could it flex on when the rainy season starts? So not waiting to confirm its ended, but saying once it starts, we know that the growing season is X days/months long, so we monitor for this period in this area knowing that growing has started.

Reply:
Yeah nice thought! I had a conversation with @PaulineNimo before the holidays related to this and the rainy season length, and she had some really nice ideas on this!